src/pages/docs/en/basics/local.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Developing Locally
+section: Basics
+position: 3
+hidden: false
+layout: "@docs"
+---
+
+## Starting a Development Server
+
+Once you've created a Space project, you can start developing your app locally using the Space CLI's `space dev` command.
+
+```bash
+space dev --open # start a local development server and open the app in your browser
```

Comment:
I would use the most simplest version of the command here and mention the open flag further down:

```suggestion
space dev
```